pkg/cloudvolume/logout_test.go

```diff
@@ -0,0 +1,89 @@
+// (C) Copyright 2019 Hewlett Packard Enterprise Development LP.
+
+package cloudvolume
+
+import (
+	"errors"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/HewlettPackard/hpecli/internal/platform/context"
+)
+
+func init() {
+	context.DefaultDBOpenFunc = context.MockOpen
+}
+
+func TestLogoutHostPrefixAdded(t *testing.T) {
+	// // clear everything from the mock store
+	context.MockClear()
+
+	mux := http.NewServeMux()
+	server := httptest.NewTLSServer(mux)
+	mux.HandleFunc("/rest/login-sessions", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
I'm guessing that since we don't make client REST calls - we don't need to setup the server and response here